cfg/cfg.go

```diff
@@ -90,12 +90,16 @@ func init() {
 	defaultPodSelector, isFound := os.LookupEnv("POD_SELECTOR")
 	if !isFound {
 		workspaceID := os.Getenv("CHE_WORKSPACE_ID")
-		if workspaceID == "" {
-			logrus.Fatal("unable to get current workspace id. Configure custom pod selector or che workspace id")
+		if workspaceID != "" {
```

Comment:
I think it should be reverse ( first dev workspace ) as I think we still had cheworkspace id with de workspace operqtor

Reply:
Makes sense. Actually, it would be a bug if CHE_WORKSPACE_ID and DEVWORKSPACE_ID would have different values on DevWorkspace operator side.

Reply:
done